nixos/modules/services/misc/mediatomb.nix

```diff
@@ -206,83 +257,129 @@ in {
 
       dataDir = mkOption {
         type = types.path;
-        default = "/var/lib/mediatomb";
+        default = "/var/lib/${name}";
         description = ''
-          The directory where mediatomb stores its state, data, etc.
+          The directory where ${name} stores its state, data, etc.
```

Comment:
I'd call this "packageName" or something. With just "name" I would expect it to refer to the service, something like "mediatomb-service".

Reply:
I won't rename name because i don't want to change too much the existing part.
I settled on using {serverName} in the description, does that sound good enough?